controllers/hyperconverged/hyperconverged_controller_test.go

```diff
@@ -401,7 +401,7 @@ var _ = Describe("HyperconvergedController", func() {
 					Message: reconcileCompletedMessage,
 				})))
 
-				verifySystemHealthStatusError(foundResource)
+				verifySystemHealthStatusError(foundResource, "SSPConditions")
```

Comment:
why here we have "" and in lines 195/317 we don't have?

Reply:
it is only used once, that's why it uses no variable